### PR TITLE
DHFPROD-1745: Primary key is not displayed on mapping entity table

### PR DESCRIPTION
**Pull request by brucean52**

### CODE REVIEW

File: quick-start/src/main/ui/app/shared/components/mappings/map-ui.component.ts [hunk 1]
```diff
@@ -357,4 +357,13 @@ export class MapUiComponent implements OnChanges {
     return _.includes(this.chosenEntity.definition.pii, name);
   }
 
+    /**
+   * Is an entity property personally identifiable information?
```
Comment (wooldridge) [inline quick-start/src/main/ui/app/shared/components/mappings/map-ui.component.ts]: Should reference PK not PII.

**Comment by ayuwono** [comment]
I'm testing the UI, it seems that something goes wrong with the map if you have multiple entities. Will test around more and update the finding

**Comment by ayuwono** [comment]
If I created two mappings from two entities. It seems that after creating the second mapping, I can't access the first mapping. It's weird that it seems that it's gone (I can't see it on the screen)

**Comment by ayuwono** [comment]
Let me try it again

**Pull request merged**